modules/about-administrator-perspective.adoc

```diff
@@ -10,6 +10,8 @@ The *Administrator* perspective enables you to view the cluster inventory, capac
 
 Cluster administrators can also open an embedded command-line terminal instance with the web terminal Operator in {product-title} 4.7 and later.
 
+include::snippets/unified-perspective-web-console.adoc[]
```

Comment:
You might want to consider moving the snippet so that you don't have two admonitions right next to each other. 
SSG: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions